jdl/validators/jdl-with-application-validator.js

```diff
@@ -107,7 +111,7 @@ function createValidator(jdlObject, logger = console, options = {}) {
   function checkForFieldErrors(entityName, jdlFields, jdlApplication) {
     const validator = new FieldValidator();
     const filtering =
-      jdlApplication.getConfigurationOptionValue('databaseType') === 'sql' &&
+      jdlApplication.getConfigurationOptionValue('databaseType') === DatabaseTypes.SQL &&
```

Comment:
Destructuring like everywhere els?

Reply:
I don't understand. Should i use === SQL instead of DatabaseTypes.SQL?

Reply:
Yes, totally
```
import { SQL } = require('../../jdl/jhipster/database-types');
...
jdlApplication.getConfigurationOptionValue('databaseType') === SQL && ...
```